src/bezpath.rs

```diff
@@ -96,8 +102,31 @@ impl VelloVector for VelloBezPath {
 
         path
     }
+
+    fn border_translation(&self, time: f64) -> DVec2 {
+        let elements = self.path.elements();
+        let index_f = (time * elements.len() as f64).max(1.0) - 1.0;
+        let index = index_f as usize;
+        let path = &elements[index..=index - (time == 1.0) as usize + 1];
+
+        // TODO: could use reflection or meta stuff here but match works for now
+        let current = path[0].end_point().unwrap_or_default().to_vec2();
+        let next = path[path.len() - 1];
+        let next = match next {
+            PathEl::LineTo(p) => p,
+            PathEl::CurveTo(_a, _b, _c) => todo!(), // should be implemented upstream kurbo
+            PathEl::QuadTo(_a, _b) => todo!(),      // should be implemented upstream kurbo
```

Comment:
kurbo should implement interpolation of bezier curves not us (but if we need to until kurbo does  than its fine i guess).

Reply:
see `interp_pathel` comments